src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -898,7 +898,10 @@ chooseLedgerEntryType(Json::Value const& params)
                  {jss::signer_list, ltSIGNER_LIST},
                  {jss::state, ltRIPPLE_STATE},
                  {jss::ticket, ltTICKET},
-                 {jss::nft_offer, ltNFTOKEN_OFFER}}};
+                 {jss::nft_offer, ltNFTOKEN_OFFER},
```

Comment:
Can we add `bridge` to this list as well?

Reply:
Fixed in 1831398345 `[fold] add bridge type to AccountObjects RPC`